genienlp/util.py

```diff
@@ -759,9 +755,18 @@ def load_config_json(args):
             'reduce_metrics',
             'database_dir',
         ]
+        # these are true/ false arguments
+        overwrite_actions = [
+            'plot_heatmaps',
```

Comment:
Why do you want to copy "plot_heatmaps" from training? It doesn't make sense...

Reply:
fixed